admin/tabs/general/alertBanner/AlertBannerPanel.ts

```diff
@@ -155,6 +160,14 @@ const formPanel = hoistCmp.factory<AlertBannerModel>(({model}) => {
             ]
```

Comment:
Have we ensured that all of these buttons/menus are not available to HOIST_ADMIN_READER?

Reply:
We should be able to globally disable the entire FormModel.  Think that is probably a simpler way to do this.